iidm/iidm-api/src/main/java/com/powsybl/iidm/network/Network.java

```diff
@@ -661,12 +661,22 @@ default HvdcLine getHvdcLine(HvdcConverterStation converterStation) {
     HvdcLineAdder newHvdcLine();
 
     /**
-     * Get a equipment.
+     * Get an equipment by its ID.
      *
      * @param id the id of the equipment
      */
     Identifiable<?> getIdentifiable(String id);
 
+    /**
+     * Get an equipment by its ID or eventually an alias.
+     *
+     * @param idOrAlias the id or an alias of the equipment
+     * @param searchInAlias if true then searches in the alias and the IDs. If false, searches only by ID.
+     */
+    default Identifiable<?> getIdentifiable(String idOrAlias, boolean searchInAlias) {
+        throw new UnsupportedOperationException();
+    }
+
```

Comment:
Do we really need this method: what is the risk to change the behavior of the `getIdentifiable(String id)` if we consider aliases like unique IDs. At the end, all the user code would use this new method, don't they?

Reply:
I do not know if it may be interesting to search objects by ID uniquely.

Reply:
As searching also with alias could impact performance, it might be interesting to keep the possibility to only seach with primary id